CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -1083,7 +1077,7 @@ namespace mu2e {
           }
         }
       }
-      _tcHits[i].uselessTripletSeed = uselessSeed;
+      if (i<_tcHits.size()) {_tcHits[i].uselessTripletSeed = uselessSeed;}
```

Comment:
is the condition i>=_tcHits.size() a sign of a logical error?  Or can it occur through the normal, correct processing of the preceeding logic?   If the former, this fix is inadequate, merely covering over an error not fixing it.  If the later is true please add a comment describing the situation by which i might legitimately be >= size()

Reply:
I asked the same question and he answered above.

Reply:
I don't see either your question or the answer.  Perhaps that was in email?

Reply:
Oh I see, that conversation was collapsed as 'resolved'.

Reply:
I just unresolved it